dmf/scripts/mods/dmf/modules/core/localization.lua

```diff
@@ -91,13 +91,12 @@ end
 -- ##### DMF internal functions and variables #########################################################################
 -- ####################################################################################################################
 
--- Handles the return of global localize text_ids
-dmf:hook(_G, "Localize", function (func, text_id, ...)
+dmf:hook(LocalizationManager, "localize", function (func, self, text_id, ...)
```

Comment:
Localize is just a wrapper around LocalizationManager:localize so this catches both uses